packaging/Dockerfile.centos8-nmstate-dev

```diff
@@ -1,12 +1,4 @@
-# This Dockerfile is based on the recommendations provided in the
```

Comment:
Why has this been removed? It is not correct anymore?

Reply:
Not needed.